### ISSUE TITLE
Restructure Processsors and Collectors docs

### ISSUE THREAD

**Issue by georgiana-b** [opentrials/opentrials#631]
We currently have separate docs for [Processors](https://github.com/opentrials/processors/tree/master/docs) and [Collectors](https://github.com/opentrials/collectors/tree/master/docs). The main README and Overview has been updated but the docs for specific collectors/processors are outdated or missing. IMO this proves our current setup needs improvements.

I propose to move all the collector/processor specific docs into the collector/processor folder.

 Ex. Move the docs for EUCTR collector from `/collectors/docs/collectors/euctr.md` to `collectors/euctr/README.md`

Reasons:
- it might remind developers to add/update them
- it will make it easier for people who read the code on GitHub since it will  be rendered on https://github.com/opentrials/collectors/tree/master/collectors/euctr

Of course, any other ideas are welcomed.

**Comment by nightsh** [comment]
I agree to that change, as suggested in https://github.com/opentrials/docs/pull/9#discussion_r94748144
